docs/2022-07-18-JSON-RPC-Server-API.md

```diff
@@ -333,18 +333,37 @@ Same as for execute
   "id": 1,
   "method": "add-module",
   "params": {
-    "name": "MODULE-NAME",
-    "module": "<plain text>"
+    "module": "module MODULE-NAME endmodule []"
   }
 }
 ```
 
-* `name` is the name of the main module.
-* `module` is module represented in KORE.
+* `module` is a module represented in textual KORE format. The module may import modules that have been loaded or added earlier.
+
+### Error Response:
+
+If the textual KORE in `module` is syntactically wrong, the response will use the error code
+`Could not parse pattern` with the parse error in the `data` field (also see below).
+
```

Comment:
@tothtamas28 could you please have a look at the modified description?
The request type has changed (`name` field omitted, parsing a _single_ module now), so this is what I made of your pull request #3535

Reply:
Looks good to me.